### ISSUE TITLE
Cycle all effects and colors

### ISSUE THREAD

**Issue by vicv0** [Aircoookie/WLED#1999]
Is there a way to have my leds just cycle through all the available effects with random colors? Like every minute change to the effect

**Comment by blazoncek** [comment]
Yes, but is a bit tricky and requires creating API presets and playlists.

**Comment by blazoncek** [comment]
Please refer to [WLED docs](https://kno.wled.ge) for information on presets and API.
